src/tests/createHeadlessForm.test.js

```diff
@@ -8,7 +8,9 @@ import {
   JSONSchemaBuilder,
   schemaInputTypeText,
   schemaInputTypeRadioDeprecated,
-  schemaInputTypeRadio,
+  schemaInputTypeRadioString,
```

Comment:
FYI @Arcombe-Remote , I've renamed the PR title to follow conventional commits :) fix(select/radio): Add support for options with boolean type (#89)

Reply:
🎉 Version `0.11.3-beta.0` published!